src/supplemental/util/platform.c

```diff
@@ -44,7 +44,6 @@ nng_thread_create(nng_thread **thrp, void (*func)(void *), void *arg)
 	if ((thr = NNI_ALLOC_STRUCT(thr)) == NULL) {
 		return (NNG_ENOMEM);
 	}
-	memset(thr, 0, sizeof(*thr));
```

Comment:
Another leftover... :-) Originally NNI_ALLOC_STRUCT() didn't promise to clear memory like it does now.

Reply:
Yeah, I searched the whole project with keyword **NNI_ALLOC_STRUCT** and found two residuals~